### PR TITLE
Bulk upload positive

### PR DESCRIPTION
**Pull request by jzhang-gp**
Bulk upload page for companies and labels with the same annotation value.

### CODE REVIEW

File: backend/annotations.py [hunk 1]
```diff
@@ -26,6 +28,69 @@ def index():
     return render_template('annotations/index.html')
 
 
+@bp.route('/bulk_v2', methods=['GET'])
+def bulk_v2():
```
Comment (edu-gp) [inline backend/annotations.py:32]: Can we use a more descriptive name than `bulk_v2`?

File: backend/annotations.py [hunk 2]
```diff
@@ -104,11 +145,52 @@ def bulk_post():
...
 
     except Exception as e:
+        logging.error(e)
         flash(str(e))
         return render_template('annotations/bulk.html')
     else:
         return redirect(redirect_to)
+
+
+def __upsert_annotations(entity_type, entity, label, user, value):
```
Comment (edu-gp) [inline backend/annotations.py:159]: 1. Please add test to `__upsert_annotations`
2. Would it be ok to use 1 underscore? e.g. `_upsert_annotations`

File: backend/annotations.py (hunk 1 above)
Comment (jzhang-gp) [reply backend/annotations.py:32]: Done. Changed to `bulk_upload_positive_annotations`.

File: backend/annotations.py (hunk 2 above)
Comment (jzhang-gp) [reply backend/annotations.py:159]: Done.

**Pull request merged**